homeassistant/components/broadlink/switch.py

```diff
@@ -336,3 +339,75 @@ async def _async_send_packet(self, packet):
             _LOGGER.error("Failed to send packet: %s", err)
             return False
         return True
+
+
+class BroadlinkBG1Slot(BroadlinkSwitch):
+    """Representation of a Broadlink BG1 slot."""
+
+    def __init__(self, device, slot):
+        """Initialize the switch."""
+        super().__init__(device, 1, 0)
+        self._slot = slot
+        self._state = self._coordinator.data[f"pwr{slot}"]
+        self._device_class = DEVICE_CLASS_OUTLET
+
+    @property
+    def unique_id(self):
+        """Return the unique id of the slot."""
+        return f"{self._device.unique_id}-{self._slot}"
+
+    @property
+    def name(self):
+        """Return the name of the switch."""
+        return f"{self._device.name} {'left' if self._slot == 1 else 'right'}"
```

Comment:
```python3
        return f"{self._device.name} S{self._slot}"
```

Reply:
IMO as there are only 2 "slots", "left" and "right" made more sense. But OK.

Reply:
I understand, but we need to think about extendability. Left and right doesn't scale.